court_scraper/base/_search_page_mixin.py

```diff
@@ -0,0 +1,76 @@
+from ._last_date import LastDate
+
+class SearchPageMixIn():
```

Comment:
This is the second library of code that provides the case discovery tool. It does not conflict with master.